pkg/server/sandbox_list_test.go

```diff
@@ -63,6 +63,10 @@ func TestToCRISandbox(t *testing.T) {
 			state:         sandboxstore.StateNotReady,
 			expectedState: runtime.PodSandboxState_SANDBOX_NOTREADY,
 		},
+		"sandbox state unknown": {
+			state:         sandboxstore.StateUnknown,
+			expectedState: runtime.PodSandboxState_SANDBOX_NOTREADY,
```

Comment:
from the state machine it looks like this should be expected ready? please confirm.

Reply:
In internal state machine, `StateUnknown` is just state unknown.
However, in CRI there is only `NOTREADY` state, and it is good enough for unknown handling:
1) Kubelet will not continue using the sandbox, and will restart it.
2) Before restart kubelet will always try to stop the previous `NOTREADY` sandbox until success.